graph/connected_components.cpp

```diff
@@ -1,29 +1,87 @@
+/**
+ *
+ * \file
+ * \brief [Connected Components
+ * (Connected
+ * Components)](https://en.wikipedia.org/wiki/Component_(graph_theory))
+ *
+ * \author [Ayaan Khan](http://github.com/ayaankhan98)
```

Comment:
not fair to the original author. Include the original author's GitHub handle first and then yours.

Reply:
@kvedala please see the PR description, i am not doing unfair with anyone, on 16th May, 2020 i had implemented this file but at that time there is no one to look forward for the documentation so this was approved by cclause at that time, so i decided to improve on this and thus added the docs. i think there is no need to get credit from other's work in #888 i added docs but those files are not implemented by me so i left the author as blank in on of the file of PR #888 whereas in other file @Panquesito7 told me the author name by tracking the git commits so i mentioned the name of original author.